hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -297,6 +297,9 @@ enum Status {
 
     USER_MISMATCH = 54; // Error code when requested user name passed is
     // different from remote user.
+
+    INVALID_PART = 55;
```

Comment:
Minor: can you add a 1 line comment to describe the two errors like `USER_MISMATCH`?